code/modules/SCP/SCPs/SCP-914.dm

```diff
@@ -0,0 +1,251 @@
+#define CLOCKMODE_ROUGH 1
+#define CLOCKMODE_COARSE 2
+#define CLOCKMODE_ONEONE 3
+#define CLOCKMODE_FINE 4
+#define CLOCKMODE_VERYFINE 5
+
+/* SCP 914 - Clockwork mechanism
+Put things into INTAKE booth, click on panel in the middle - take resulting product from OUTPUT booth
+TO ADD RECIPES:
+1. find an item you want to use.
+2. put following lists to item's definition:
+  rough = list(<list of items for that mode>)
+  (required) coarse = list(...)
+  (required) oneone = list(...)
+  (required) fine = list(...)
+  veryfine = list(...)
+You may use only one list, but then selected item would be transformed only on mode where corresponding list is defined.
+By default Rough/Very Fine will process item on Coarse/Fine twice, define coarse/veryfine lists to specify any desired transformation
+Proc /atom/scp914_act(var/mode) can be overriden to define any additional effects by your desire (for example, transfer mind from initial mob to processed one)
+*/
+/datum/scp/SCP_914
+  name = "SCP-914"
+  designation = "914"
+  classification = SAFE
+  component = /datum/component/scp/SCP_914
+
+/datum/component/scp/SCP_914
+	signal_procs = list(COMSIG_PARENT_ATTACKBY = .proc/transform)
+	var/clockwork = FALSE
+
+/datum/component/scp/SCP_914/proc/transform(obj/item/W, mob/user, var/click_params)
+  if(!W || clockwork)
+    return
```

Comment:
Is GitHub failing to display this or is this file's indentation messed up completely?